frontend/src/components/BudgetLineItems/AllBudgetLinesTable/AllBudgetLinesTable.hooks.js

```diff
@@ -42,4 +42,20 @@ function useAllBudgetLinesTable(budgetLines) {
     return { showModal, setShowModal, modalProps, handleDeleteBudgetLine };
 }
 
+export const useSetSortConditions = () => {
```

Comment:
may consider moving this logic into a reusable hook 
- maybe in the `Table.hooks`

Reply:
took care of this